tensorflow/workspace.bzl

```diff
@@ -83,6 +84,49 @@ temp_workaround_http_archive = repository_rule(
 )
 
 
+def _run_cmd(repo_ctx, cmd, check=True):
+  if _is_windows(repo_ctx):
+    bazel_sh = _get_env_var(repo_ctx, "BAZEL_SH")
+    if not bazel_sh:
+      fail("BAZEL_SH environment variable is not set")
+    cmd = [bazel_sh, "-c", " ".join(cmd)]
+  if check:
+    _execute_and_check_ret_code(repo_ctx, cmd)
+  else:
+    return repo_ctx.execute(cmd, timeout=10)
+
+def _unzip(repo_ctx, zip_file, strip_prefix):
+  _run_cmd(repo_ctx,
+           ["unzip", repo_ctx.path(zip_file),
+            "-d", repo_ctx.path("_archive")])
+  if strip_prefix:
+    p = repo_ctx.path("_archive/" + strip_prefix)
+    for f in p.readdir():
+      _run_cmd(repo_ctx, ["mv", f, repo_ctx.path(".")])
+    _run_cmd(repo_ctx, ["rm", "-rf", repo_ctx.path("_archive")])
+
+def _unsafe_unzip_http_archive_impl(repo_ctx):
+  repo_ctx.symlink(repo_ctx.attr.build_file, "BUILD")
+  repo_ctx.download(repo_ctx.attr.urls, "_archive.zip", repo_ctx.attr.sha256)
+  _unzip(repo_ctx, "_archive.zip", repo_ctx.attr.strip_prefix)
+  _run_cmd(repo_ctx, ["rm", repo_ctx.path("_archive.zip")])
+  repo_ctx.symlink("/usr/include", "local_include/")
+
+
+# This performs an unsafe unzip operation - since zip files may have
+# relative and absolute paths, it should only be executed on trusted files.
```

Comment:
@meteorcloudy Can bazel handle zip files with symlinks in windows now?
Would this break windows bazel build (not currently run in presubmits)